src/bin/common/parse_command.ml

```diff
@@ -0,0 +1,1157 @@
+
+  let interpretation =
+    let doc =
+      "Experimental support for counter-example generation. Possible \
+       values are 1, 2, or 3 to compute an interpretation before returning \
+       Unknown, before instantiation, or before every decision or \
+       instantiation. A negative value (-1, -2, or -3) will disable \
+       interpretation display. Note that $(b, --max-split) limitation will \
+       be ignored in model generation phase." in
+    let docv = "VAL" in
+    Arg.(value & opt int (get_interpretation ()) &
+         info ["interpretation"] ~docv ~docs ~doc) in
+
+  let model =
+    let doc = Format.sprintf
+        "Experimental support for models on labeled terms. \
+         $(docv) must be %s. %s shows a complete model and %s shows \
+         all models."
+        (Arg.doc_alts ["none"; "default"; "complete"; "all"])
+        (Arg.doc_quote "complete") (Arg.doc_quote "all") in
+    let docv = "VAL" in
+    Arg.(value & opt model_conv MNone & info ["m"; "model"] ~docv ~doc) in
+
+  let unsat_core =
+    let doc = "Experimental support for unsat-cores." in
+    Arg.(value & flag & info ["u"; "unsat-core"] ~doc) in
+
+  let output_format =
+    let doc =
+      Format.sprintf
+        "Answer unsat/sat/unknown instead of Valid/Invalid/I don't know. \
+         $(docv) must be %s. It must be noticed that not specifying an output \
+         format will let Alt-Ergo set it according to the input file's \
+         extension."
+        (Arg.doc_alts [ "native"; "smtlib" ])
+    in
+    let docv = "FMT" in
+    Arg.(value & opt (some format_conv) None & info ["o"; "output"] ~docv ~doc)
+  in
+
+  Term.(ret (const mk_output_opt $
+             interpretation $ model $ unsat_core $ output_format
+            ))
+
+let parse_profiling_opt =
+
+  let docs = s_profiling in
+
+  let cumulative_time_profiling =
+    let doc =
+      "Record the time spent in called functions in callers" in
+    Arg.(value & flag & info ["cumulative-time-profiling"] ~docs ~doc) in
+
+  let profiling =
+    let doc =
+      "Activate the profiling module with the given frequency. \
+       Use Ctrl-C to switch between different views and \\\"Ctrl \
+       + AltGr + \" to exit." in
+    let docv = "DELAY" in
+    Arg.(value & opt (some float) None & info ["profiling"] ~docv ~docs ~doc) in
+
+  let profiling_plugin =
+    let doc = "Use the given profiling plugin." in
+    let docv = "PGN" in
+    Arg.(value & opt string (get_profiling_plugin ()) &
+         info ["profiling-plugin"] ~docv ~docs ~doc) in
+
+  let get_verbose =
+    let doc = "Set the get_verbose mode." in
+    Arg.(value & flag & info ["v"; "verbose"] ~doc) in
+
+  Term.(ret (const mk_profiling_opt $
+             cumulative_time_profiling $ profiling $
+             profiling_plugin $ get_verbose
+            ))
+
+let parse_quantifiers_opt =
+
+  let docs = s_quantifiers in
+
+  let greedy =
+    let doc = "Use all available ground terms in instantiation." in
```

Comment:
Maybe add something briefly specifying what is the default without this ?

Reply:
I don't know the default.